binder/environment.yml

```diff
@@ -0,0 +1,61 @@
+name: pangeo
+channels:
+ - conda-forge
+dependencies:
+ - python=3.7*
+ - pangeo-notebook=2020.04.28
```

Comment:
Bump this to the latest (`2020.05.10`)? Also this probably needs to be kept in sync with the docker version.

Reply:
Could we tie this repo to the pangeo-bot to avoid this manual step?

Reply:
I'm not sure, hopefully @scottyhq can answer. I'm guessing this would be easiest if it were in the `pangeo-docker-images` repo, but it may be doable here as well.

How often do we want the `default-binder` image to be updated? Each time there's an update to one of the core packages?